src/Symfony/Bundle/FrameworkBundle/Templating/Helper/TranslatorHelper.php

```diff
@@ -39,6 +43,10 @@ public function trans($id, array $parameters = array(), $domain = 'messages', $l
      */
     public function transChoice($id, $number, array $parameters = array(), $domain = 'messages', $locale = null)
     {
+        if (null === $this->translator) {
+            return strtr($id, $parameters);
```

Comment:
The plural rules should still be resolved here, the trait provided in contracts should be used for that purpose.

Reply:
OR remove strtr entirely as in #24358 !

Reply:
In fact, TwigBridge also needs the fix to handle plural, see  #24514